code/preprocess.py

```diff
@@ -37,7 +38,7 @@ def preprocess_test(domain):
 
     for text, label in zip(f1, f2):
         label = label.strip()
-        if domain == 'restaurant' and label not in ['Food', 'Staff', 'Ambience']:
+        if domain == 'speaker' and label not in ['sound', 'durability', 'design', 'price']:
```

Comment:
it's ok to use this code for other datasets, but maybe you could rewrite it to be more universal?